geemap/core.py

```diff
@@ -469,9 +469,6 @@ def __init__(self, **kwargs):
         kwargs = self._apply_kwarg_defaults(kwargs)
         super().__init__(**kwargs)
 
-        if len(self.layers) > 0 and self.layers[0].name == "OpenStreetMap.Mapnik":
-            self.layers[0].name = "OpenStreetMap"
```

Comment:
I think this rename needs to be carried over to the normal geemap.Map constructor since it may be expected in other parts of geemap:
https://github.com/search?q=repo%3Agee-community%2Fgeemap+OpenStreetMap&type=code

Reply:
geemap and geemap core have different basemap names for the curated basemaps (default, hybrid, etc.). That mapping in core was (according to Qiusheng) to reduce the length of the map name displayed in the layer manager. It shouldn't be necessary.

I'm testing geemap (non-core) now, and basemap selection is WAI

Reply:
Nevermind! I am incorrect, the base map implementations use different keys.

Reply:
Awesome, thanks for clarifying!

Reply:
It seems the rename works fine with geemap.py, but breaks core.py. I am looking into it now.